### DISCUSSION TITLE
VRoid SDK 自身のバージョンを表す定数またはAPIなどありますでしょうか？

### DISCUSSION THREAD

**Discussion by fantom1x** [pixiv/vroid-sdk-developers#72] [Q&A]
UniVRM で言う「VRMVersion」(static class の const)
Unity で言う「Application.unityVersion」(API)
のような、VRoid SDK 自身のバージョンを表す定数などを取得する方法はありますでしょうか？

できれば、アプリやエディタ起動中に、常に利用バージョンを確認(自動取得)できるようなものが欲しいです(何でも良いです)。

**Comment by nyamadan** [comment]
大変お待たせして申し訳ございません。

`Pixiv.VroidSdk.Version.CurrnetVersion` を使用すれば現在使用中のVRoid SDKのバージョンを取得可能です。

https://developer.vroid.com/sdk/docs/0.1.3/Pixiv.VroidSdk.Version.html#Pixiv_VroidSdk_Version_CurrentVersion

**Reply by fantom1x** [reply]
お答え有難うございます！定数があったのですね。ぜひ利用させて頂きます！m(_ _)m
